src/browser.ts

```diff
@@ -21,6 +21,17 @@ import { Download } from './download';
 import type { BrowserServer } from './server/browserServer';
 import { Events } from './events';
 import { InnerLogger, Log } from './logger';
+import * as types from './types';
+
+export type BrowserOptions = {
+  logger: InnerLogger,
+  downloadsPath: string,
+  persistent?: boolean,
+  slowMo?: number,
+  headless?: boolean,
```

Comment:
Does this default to true? In public api it does. It'll probably bite us, reverse?

Reply:
Good catch. Fixed.